src/main/java/net/bramp/ffmpeg/Preconditions.java

```diff
@@ -41,7 +42,7 @@ public static String checkNotEmpty(String arg, @Nullable Object errorMessage) {
    */
   public static URI checkValidStream(URI uri) throws IllegalArgumentException {
     String scheme = checkNotNull(uri).getScheme();
-    scheme = checkNotNull(scheme, "URI is missing a scheme").toLowerCase();
+    scheme = Ascii.toLowerCase(checkNotNull(scheme, "URI is missing a scheme"));
```

Comment:
Is there a reason between the two?
Based on the internals, I prefer the Java implementation, as it handles different encoding systems better.
Although this is a scheme, which should be okay with Ascii.

Reply:
This was https://errorprone.info/bugpattern/StringCaseLocaleUsage, where without specifications a locale the behaviour is different in different platforms.

Since we know the scheme must be ASCII, this ASCII.toLowerCase is just a convenient way to express that.